src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write your code here
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+const population = document.getElementsByClassName('population');
+
+const data = [];
+
+for (const el of population) {
+  const tmp = el.innerText;
+
+  data.push(parseInt(tmp.replaceAll(',', '')));
+}
+
+const totalData = data.reduce((el, el2) => el + el2, 0);
```

Comment:
The variable names `el` and `el2` are not descriptive. Consider renaming them to reflect their purpose, like `accumulator` and `currentValue`.